test/modules/layers/test_quantization.py

```diff
@@ -38,11 +38,14 @@ def setUp(self):
         )
 
         self.vq = Quantization(
-            num_embeddings=self.num_embeddings, embedding_dim=self.embedding_dim
+            num_embeddings=self.num_embeddings,
+            embedding_dim=self.embedding_dim,
+            decay=0.5,
```

Comment:
super nit but I would use a different value than 0.5 as it can mask errors with decay and 1-decay mixups

Reply:
good point